mllib/src/test/scala/org/apache/spark/mllib/classification/NaiveBayesSuite.scala

```diff
@@ -116,6 +116,92 @@ class NaiveBayesSuite extends SparkFunSuite with MLlibTestSparkContext {
     }
   }
 
+  def validatePredictionsProbabilities(predictionsProbabilities: Seq[Array[Double]],
+                                       input: Seq[LabeledPoint],
+                                       modelType: String = Multinomial) = {
+    predictionsProbabilities.foreach { probabilities =>
+      val sum = probabilities.sum
+      // Check that prediction probabilities sum up to one
+      // with an epsilon of 10^-2
+      assert(sum > 0.99 && sum < 1.01)
```

Comment:
We have a syntax like `assert(sum ~== 1.0 relTol 0.01)` for this. I think the bound should be tighter here. I would not expect probabilities to be 1.001 or 1.0001.